templates/microprofile-openliberty/files/README.md

```diff
@@ -0,0 +1,26 @@
+# Open Liberty for Platform.sh
+
+This project provides a starter kit for Open Liberty Eclipse MicroProfile projects hosted on Platform.sh.
```

Comment:
This line should also be it's own first <p> tag in .platform.template.yaml

Reply:
Fixed, thank you @chadwcarlson